<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="topMargin">
     <number>15</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>15</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QScrollArea {
	border: 5px solid teal;
}</string>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustToContents</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>266</width>
           <height>521</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="labelAlignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="formAlignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
          <property name="leftMargin">
           <number>11</number>
          </property>
          <property name="rightMargin">
           <number>11</number>
          </property>
          <item row="0" column="0" colspan="2">
           <widget class="QLabel" name="label_2">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel {
	font-weight: bold;
}</string>
            </property>
            <property name="text">
             <string>Формат записки</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Лист А4 - 210х297мм</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QRadioButton" name="r_b_format_1">
            <property name="text">
             <string>4шт 75мм*210мм</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroupFormat</string>
            </attribute>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QRadioButton" name="r_b_format_2">
            <property name="text">
             <string>6шт 70мм*148мм</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroupFormat</string>
            </attribute>
           </widget>
          </item>
          <item row="4" column="0" colspan="2">
           <widget class="QRadioButton" name="r_b_format_3">
            <property name="text">
             <string>4шт  105мм*148мм</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroupFormat</string>
            </attribute>
           </widget>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="QRadioButton" name="r_b_format_4">
            <property name="text">
             <string>2шт  105мм*296мм</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroupFormat</string>
            </attribute>
           </widget>
          </item>
          <item row="6" column="0" colspan="2">
           <widget class="QRadioButton" name="r_b_format_5">
            <property name="text">
             <string>3шт 99мм*210мм</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroupFormat</string>
            </attribute>
           </widget>
          </item>
          <item row="7" column="0" colspan="2">
           <widget class="QLabel" name="label_4">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel {
	font-weight: bold;
}</string>
            </property>
            <property name="text">
             <string>Кол-во имен в записке</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="8" column="0" colspan="2">
           <widget class="QLabel" name="label_5">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>(1-50)</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="9" column="0" colspan="2">
           <widget class="QSlider" name="horizontalSlider">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>50</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="10" column="0" colspan="2">
           <widget class="QLabel" name="label_6">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
              <kerning>true</kerning>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel {
	font-weight: bold;
}</string>
            </property>
            <property name="text">
             <string>Тип записки</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="11" column="0" colspan="2">
           <widget class="QRadioButton" name="r_b_type_1">
            <property name="text">
             <string>О ЗДРАВИИ</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroupType</string>
            </attribute>
           </widget>
          </item>
          <item row="12" column="0" colspan="2">
           <widget class="QRadioButton" name="r_b_type_2">
            <property name="text">
             <string>О УПОКОЕНИИ</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroupType</string>
            </attribute>
           </widget>
          </item>
          <item row="13" column="0" colspan="2">
           <widget class="QRadioButton" name="r_b_type_3">
            <property name="text">
             <string>УНИВЕРСАЛЬНАЯ</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroupType</string>
            </attribute>
           </widget>
          </item>
          <item row="15" column="0" colspan="2">
           <widget class="QLabel" name="label_10">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel {
	font-weight: bold;
}</string>
            </property>
            <property name="text">
             <string>Наполнение записок</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="16" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>20</number>
            </property>
            <item>
             <widget class="QRadioButton" name="r_b_fill_1">
              <property name="sizeIncrement">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Пустые</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroupFilling</string>
              </attribute>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="r_b_fill_2">
              <property name="sizeIncrement">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>Добавить имена</string>
              </property>
              <attribute name="buttonGroup">
               <string notr="true">buttonGroupFilling</string>
              </attribute>
             </widget>
            </item>
           </layout>
          </item>
          <item row="19" column="0" colspan="2">
           <widget class="QPushButton" name="pushButton">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <weight>75</weight>
              <bold>true</bold>
              <strikeout>false</strikeout>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	margin-top: 5px;
	margin-bottom: 5px;
	margin-left: 25px;
	margin-right: 25px;
	padding: 5px;
	font-weight: bold;
}</string>
            </property>
            <property name="text">
             <string>Выберите цвет рамки</string>
            </property>
           </widget>
          </item>
          <item row="20" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>15</number>
            </property>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="commitButton">
              <property name="styleSheet">
               <string notr="true">QPushButton {
	background-color: rgb(66, 142, 255);
	border: 2px solid rgb(69, 73, 75);
	border-radius: 5px;
	padding-right: 10px;
	padding-left: 10px;
	font-size: 18px;
	color: rgb(255, 255, 255);
}</string>
              </property>
              <property name="text">
               <string>Commit</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="18" column="0" colspan="2">
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true"/>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>20</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	margin-right: 15px;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionData">
   <property name="text">
    <string>Данные о поминаемых</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Помощь</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroupFormat"/>
  <buttongroup name="buttonGroupFilling"/>
  <buttongroup name="buttonGroupType"/>
 </buttongroups>
</ui>
